src/versionbits.h

```diff
@@ -11,11 +11,11 @@
 /** What block version to use for new blocks (pre versionbits) */
 static const int32_t VERSIONBITS_LAST_OLD_BLOCK_VERSION = 4;
 /** What bits to set in version for versionbits blocks */
-static const int32_t VERSIONBITS_TOP_BITS = 0x20000000UL;
+static const int32_t VERSIONBITS_TOP_BITS = 0x04000000UL;
```

Comment:
What are we updating these to? Perhaps we should discuss what makes sense.

Reply:
So. For the PoW changes. In order to know which algo is used to mine the block. I am taking over an addition 3 bits of the block version. Bits 29, 28, and 27. this changes the available bits for bip 9 softforks

Reply:
Ok

Reply:
Done